doc/base/classes.xml

```diff
@@ -45788,46 +45815,60 @@
 			<argument index="0" name="offset" type="Vector3">
 			</argument>
 			<description>
+				Change node position by given offset [Vector3].
 			</description>
 		</method>
 		<method name="update_gizmo">
 			<return type="void">
 			</return>
 			<description>
+				Update [SpatialGizmo] of this node.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="global_transform" type="Transform" setter="set_global_transform" getter="get_global_transform" brief="">
+			World space (global) [Transform] of this node.
 		</member>
 		<member name="rotation" type="Vector3" setter="set_rotation" getter="get_rotation" brief="">
+			Local euler rotation in radians of this node.
 		</member>
 		<member name="rotation_deg" type="Vector3" setter="set_rotation_deg" getter="get_rotation_deg" brief="">
+			Local euler rotation in degrees of this node.
 		</member>
 		<member name="scale" type="Vector3" setter="set_scale" getter="get_scale" brief="">
+			Local scale of this node.
 		</member>
 		<member name="transform" type="Transform" setter="set_transform" getter="get_transform" brief="">
+			Local space [Transform] of this node.
 		</member>
 		<member name="translation" type="Vector3" setter="set_translation" getter="get_translation" brief="">
+			Local translation of this node.
 		</member>
 		<member name="visible" type="bool" setter="set_visible" getter="is_visible" brief="">
+			Visibility of this node. Toggles if this node is rendered.
 		</member>
 	</members>
 	<signals>
 		<signal name="visibility_changed">
 			<description>
+				Emitted when node visibility changed.
 			</description>
 		</signal>
 	</signals>
 	<constants>
 		<constant name="NOTIFICATION_TRANSFORM_CHANGED" value="29" enum="">
-			Spatial nodes receive this notification with their global transform changes. This means that either the current or a parent node changed its transform.
+			Spatial nodes receive this notification when their global transform changes. This means that either the current or a parent node changed its transform.
+			In order for NOTIFICATION_TRANSFORM_CHANGED to work user first needs to ask for it, with set_notify_transform(true). There is also set_notify_local_transform(true).
```

Comment:
I would skip `There is also set_notify_local_transform(true).` part. Since I assume some kind of `NOTIFICATION_LOCAL_TRANSFORM` would be needed for this and it's not exposed at the moment(?).

Reply:
Ah forgot to delete this line xD.
Yup it is not exposed.